### ISSUE TITLE
CheatPrevention elytra block is not working properly for users on Bedrock

### ISSUE THREAD

**Issue by Angry-Pineapple3121** [SirBlobman/CombatLogX#813]
### Server Implementation

Paper

### Server Version

1.20.1

### Describe the bug

Users who are playing on Bedrock (connected through Geyser) are able to glide using Elytras while in combat. Firework rockets, however, do not work for users on Bedrock while in combat. The message from the CheatPrevention module is properly displaying "You are not allowed to use elytra during combat."

### Reproduction Steps

1. One user must be connected on a version of Bedrock through Geyser.
2. The user should enter combat while wearing an Elytra.
3. After the user has entered combat, they should be able to glide using an Elytra like normal.

### Expected behaviour

The player should be prevented from gliding with an Elytra while in combat, similarly to how it affects users on Java edition.

### Screenshots / Videos

https://github.com/SirBlobman/CombatLogX/assets/74722209/509a8942-4d23-453e-bcbf-ab76f2a428d2

### CombatLogX Version

[01:16:18 INFO]: Java Version: 17.0.10
[01:16:18 INFO]: Java Vendor: Eclipse Adoptium
[01:16:18 INFO]: Java URL: N/A
[01:16:18 INFO]: Server Version: git-Axolotl-"9b61ea9" (MC: 1.20.1)
[01:16:18 INFO]: Bukkit Version: 1.20.1-R0.1-SNAPSHOT
[01:16:18 INFO]: Minecraft Version: 1.20.1
[01:16:18 INFO]: NMS Version: 1_20_R1
[01:16:18 INFO]: Dependency Information:
[01:16:18 INFO]:  - AngelChest (not installed)
[01:16:18 INFO]:  - ASkyBlock (not installed)
[01:16:18 INFO]:  - BentoBox (not installed)
[01:16:18 INFO]:  - Citizens (not installed)
[01:16:18 INFO]:  - CMI (not installed)
[01:16:18 INFO]:  - CrackShot (not installed)
[01:16:18 INFO]:  - CrashClaim (not installed)
[01:16:18 INFO]:  - Essentials v2.20.1
[01:16:18 INFO]:  - FabledSkyBlock (not installed)
[01:16:18 INFO]:  - FeatherBoard (not installed)
[01:16:18 INFO]:  - FlagWar (not installed)
[01:16:18 INFO]:  - GriefDefender (not installed)
[01:16:18 INFO]:  - GriefPrevention (not installed)
[01:16:18 INFO]:  - HuskHomes (not installed)
[01:16:18 INFO]:  - HuskSync (not installed)
[01:16:18 INFO]:  - HuskTowns (not installed)
[01:16:18 INFO]:  - iDisguise (not installed)
[01:16:18 INFO]:  - IridiumSkyblock (not installed)
[01:16:18 INFO]:  - Kingdoms (not installed)
[01:16:18 INFO]:  - Konquest (not installed)
[01:16:18 INFO]:  - Lands (not installed)
[01:16:18 INFO]:  - LibsDisguises (not installed)
[01:16:18 INFO]:  - MarriageMaster (not installed)
[01:16:18 INFO]:  - MythicMobs v5.6.0-SNAPSHOT-6916dd67
[01:16:18 INFO]:  - PlaceholderAPI v2.11.5
[01:16:18 INFO]:  - PlayerParticles (not installed)
[01:16:18 INFO]:  - PreciousStones (not installed)
[01:16:18 INFO]:  - PremiumVanish (not installed)
[01:16:18 INFO]:  - ProtectionStones (not installed)
[01:16:18 INFO]:  - ProtocolLib v5.1.0
[01:16:18 INFO]:  - RedProtect (not installed)
[01:16:18 INFO]:  - Residence (not installed)
[01:16:18 INFO]:  - Sentinel (not installed)
[01:16:18 INFO]:  - SuperiorSkyblock2 (not installed)
[01:16:18 INFO]:  - SuperVanish (not installed)
[01:16:18 INFO]:  - Towny (not installed)
[01:16:18 INFO]:  - UltimateClaims (not installed)
[01:16:18 INFO]:  - uSkyBlock (not installed)
[01:16:18 INFO]:  - VanishNoPacket (not installed)
[01:16:18 INFO]:  - WorldGuard v7.0.9+5934e49
[01:16:18 INFO]:  - ViaVersion v4.9.3
[01:16:18 INFO]:  - BlueSlimeCore v2.9.4.377
[01:16:18 INFO]: CombatLogX by SirBlobman
[01:16:18 INFO]: Local Version: 11.4.0.1.1193
[01:16:18 INFO]: Remote Version: 11.4.0.1.1193
[01:16:18 INFO]: Enabled Expansions (9):
[01:16:18 INFO]:  - Boss Bar v17.1
[01:16:18 INFO]:  - Cheat Prevention v17.6
[01:16:18 INFO]:  - EssentialsX Compatibility v17.2
[01:16:18 INFO]:  - LuckPerms Compatibility v17.1
[01:16:18 INFO]:  - PlaceholderAPI Compatibility v17.1
[01:16:18 INFO]:  - WorldGuard Compatibility v17.2
[01:16:18 INFO]:  - Death Effects v17.0
[01:16:18 INFO]:  - End Crystal Helper v17.1
[01:16:18 INFO]:  - Logger v17.2

### Anything else?

I am aware that Bedrock compatibility through Geyser is _not_ officially supported, and if you are unable to fix the issue, that's totally okay!

The server version, Axolotl, is a Paper fork.
Geyser version 2.2.2-SNAPSHOT (git-master-112f4dd)

**Comment by SirBlobman** [comment]
Have you tried using `force-prevent-elytra: true`  in your CheatPrevention `items.yml` file?

**Comment by Angry-Pineapple3121** [comment]
Yes, `force-prevent-elytra` is set to `true`. Additionally, `force-disable-flight` in `flight.yml` is also set to `true`.

**Comment by satadrive420** [comment]
We are currently experiencing the same issue.

**Comment by yeesub** [comment]
same, any solution?

**Comment by satadrive420** [comment]
> Have you tried using `force-prevent-elytra: true` in your CheatPrevention `items.yml` file?

Hi,

I was wondering if you had a chance to revisit this issue. It is a major issue for cross-play servers.
